webui/src/components/reportabuse.jsx

```diff
@@ -0,0 +1,187 @@
+import React from 'react/lib/ReactWithAddons';
+import { Link, browserHistory } from 'react-router'
+
+import Toggle from 'react-toggle';
+import { DropdownList } from 'react-widgets';
+import { HeightAnimate, ReplaceAnimate } from './animate.jsx';
+
+//TODO: handle the submission
```

Comment:
AFAIR it wasn't clear how we should handle it. If there is new information please update me to fix it!